include/ccf/ds/openapi.h

```diff
@@ -421,6 +421,11 @@ namespace ds
       const nlohmann::json& param)
     {
       auto& params = parameters(path(document, uri));
+      for (auto& p : params)
+      {
+        if (p["name"] == param["name"])
```

Comment:
Without this, path parameters get added once per HTTP verb, causing duplicates (eg. `GET` and `DELETE` on the same url). This is "first definition wins", which I think is ok because path parameters are always otherwise identical/fully determined by their name.